pkg/operator/controller/ingress/controller.go

```diff
@@ -134,9 +134,51 @@ func New(mgr manager.Manager, config Config) (controller.Controller, error) {
 	if err := c.Watch(source.Kind(operatorCache, &configv1.Proxy{}), handler.EnqueueRequestsFromMapFunc(reconciler.ingressConfigToIngressController)); err != nil {
 		return nil, err
 	}
+	// Watch secrets in the operand namespace that are referenced as default certificates so that
+	// checkDefaultCertificate can determine whether changes to the default certificate affect upgradeability.
+	// This watch is NOT required for updating the router deployment, as its volume mount automatically
+	// handles updates to the default certificate.
```

Comment:
I was thinking it would make sense to state the **non**-reason for the watch:  It would be easy to assume that we need the watch in order to update router pods when the secret changes; however, the kubelet automatically updates the router pods' volume mount when the secret changes, so this watch is *not* needed for that purpose.  

On further thought, this might be unnecessary to state explicitly, and the lack of need for a watch to update router pods precedes any of this SHA-1 stuff, so feel free to disregard this suggestion.

Reply:
I will disregard, thanks.